h2/src/main/org/h2/command/dml/Explain.java

```diff
@@ -72,7 +72,7 @@ public ResultInterface query(int maxrows) {
         Database db = session.getDatabase();
         ExpressionColumn expr = new ExpressionColumn(db, column);
         Expression[] expressions = { expr };
-        result = new LocalResult(session, expressions, 1);
+        result = session.getDatabase().getResultFactory().create(session, expressions, 1);
```

Comment:
There is a `db` variable that should be used instead of `session.getDatabase()`.